addons/base_iban/static/src/components/iban_widget/iban_widget.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates>
+    <t t-name="base_iban.iban" t-inherit="web.CharField" owl="1">
+        <xpath expr="//input" position="attributes">
+            <attribute name="t-on-input">onInputValidateIban</attribute>
+        </xpath>
+        <xpath expr="//input" position="after">
+            <t t-if="this.state.isValidIBAN === true">
+                <i class="fa fa-check o_iban text-success"/>
```

Comment:
As I understand the logic, you will see a fa-check icon by just clicking on the line (if it's valid) - why?
Though I tried it on runbot, and I don't see anything (no even the invalid icon)

Reply:
You told me that it would be nice to have it directly when start editing. I thought you were right and implemented it.

Reply:
I'm not convinced:
by just activating the row - there is a call to the server, I don't think its necessary
nor the `fa-check`

Reply:
Let's discussed it live because I'm not sure to get your point.